src/rain.c

```diff
@@ -0,0 +1,443 @@
+#include "os.h"
```

Comment:
We're probably still not merging your PR but just for clarity, you shouldn't just remove all "my" copyright stuff but instead you should change them to valid ones pointing at yourself. I personally don't care about them a lot, but entities like Debian and Ubuntu do and we're struggling to please them.